Tiltfile

```diff
@@ -1,79 +1,95 @@
 #!/usr/bin/env python
 load('ext://restart_process', 'docker_build_with_restart')
 load('ext://color', 'color')
+load('ext://dotenv', 'dotenv')
 
 analytics_settings(True) # Enable telemetry dialogue in web UI
 disable_snapshots()      # Disable TiltCloud Snapshots
 version_settings(True)   # Enable 'new version' banner
 
-# parse Tiltfile config
-config.define_string(name="env", args=False, usage="This argument defines the build env (dev or release)")
-settings = config.parse()
-environment=settings.get('env', "dev") #dev or release
+# Load environment variables from .env file
+dotenv()
 
-# load ingresses
-k8s_yaml('./cluster/charts/fuel-local/crds/traefik-resources.yaml')
-k8s_kind('IngressRoute')
+# Build publisher image with proper configuration for Minikube
+custom_build(
+    ref='fuel-streams-publisher:latest',
+    command=['./cluster/scripts/build_publisher.sh'],
+    deps=[
+        './src',
+        './Cargo.toml',
+        './Cargo.lock',
+        './docker/fuel-streams-publisher.Dockerfile'
+    ],
+    live_update=[
+        sync('./src', '/usr/src'),
+        sync('./Cargo.toml', '/usr/src/Cargo.toml'),
+        sync('./Cargo.lock', '/usr/src/Cargo.lock'),
+        run('cargo build', trigger=['./src', './Cargo.toml', './Cargo.lock'])
+    ],
+    skips_local_docker=True,
+    ignore=['./target']
+)
 
-# disable unused image
-update_settings(suppress_unused_image_warnings=["k8s-tools:latest"])
-
-# project namespace
-namespace = "fuel-local"
-
-# load fuel helm chart
-fuel_chart_name = "fuel"
-fuel_chart_dir = "cluster/charts/fuel-local"
-fuel_values = "cluster/charts/fuel-local/values.yaml"
-k8s_yaml(helm(fuel_chart_dir, name=fuel_chart_name, namespace=namespace, values=[fuel_values], set=[]))
-
-# load nats helm chart
-nats_chart_name = "nats"
-nats_chart_dir = "cluster/charts/fuel-nats"
-nats_values = "cluster/charts/fuel-nats/values.yaml"
-k8s_yaml(helm(nats_chart_dir, name=nats_chart_name, namespace=namespace, values=[nats_values], set=[]))
-
-# load surrealdb helm chart
-surrealdb_chart_name = "surrealdb"
```

Comment:
I removed all surrealdb related items since we are not using them anymore. We can always refer to this PR if we want to revisit that.